debian/control

```diff
@@ -36,6 +40,7 @@ Package: libjxl-dev
 Architecture: any
 Section: libdevel
 Depends: libjxl (= ${binary:Version}), ${misc:Depends}
+ libhwy-dev,
```

Comment:
I don't think this is correct. The public API in libjxl-dev doesn't require any hwy header. Is there something in libjxl-dev that's currently depending on libhwy-dev?

Reply:
hmmm.. I guess libjxl.a (installed by libjxl-dev) needs libhwy.a (installed by libhwy-dev)

Reply:
Everything I try to build against libjxl-dev fails when libhwy-dev is not also installed.  Nothing else currently  uses libhwy, except libjxl.  Looks like you found the specific files involved.

Reply:
Could you post the error? Unless you are linking statically it shouldn't fail.

Reply:
autoconf just didn't detect libjxl as being installed unless libhwy was also installed.

Reply:
When I try to build the libjxl/examples folder without libhwy-dev installed, this is what cmake reports:
```
-- Checking for module 'libjxl'
--   Package 'libhwy', required by 'libjxl', not found
CMake Error at /usr/share/cmake-3.18/Modules/FindPkgConfig.cmake:545 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindPkgConfig.cmake:733 (_pkg_check_modules_internal)
  CMakeLists.txt:14 (pkg_check_modules)
```